hera_pspec/pspecdata.py

```diff
@@ -1640,6 +1640,44 @@ def Jy_to_mK(self, beam=None):
                 dset.data_array[:, :, :, j] *= factors[p][None, None, :]
             dset.vis_units = 'mK'
 
+    def trim_dset_lsts(self, lst_tol=6):
```

Comment:
Is this something that could be done with `select_common` instead?

Reply:
that operates on `UVPSpec`, if I'm not mistaken. this is supposed to operate on `UVData` objects. We could make `select_common` work on `UVData`, but it seems like its use-case is really relegated to just `UVPSpec`..